plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -38,8 +37,12 @@ type Elasticsearch struct {
 	OverwriteTemplate   bool
 	ForceDocumentID     bool `toml:"force_document_id"`
 	MajorReleaseNumber  int
-	FloatHandling       string          `toml:"float_handling"`
-	FloatReplacement    float64         `toml:"float_replacement_value"`
+	FloatHandling       string  `toml:"float_handling"`
+	FloatReplacement    float64 `toml:"float_replacement_value"`
+	UsePipeline         string  `toml:"use_pipeline"`
+	DefaultPipeline     string  `toml:"default_pipeline"`
+	PipelineTagKeys     []string
+	PipelineName        string
```

Comment:
Please add TOML tags here.

Reply:
These are just for storage in the structure and are not set from the config file, therefore there are no toml tags to add.. these are the same as `TagKeys` (and `IndexName` when setting it from tags).

Reply:
If so, please do not export those fields.